ca/client.go

```diff
@@ -1138,8 +1135,7 @@ retry:
 	}
 	var check api.SSHCheckPrincipalResponse
 	if err := readJSON(resp.Body, &check); err != nil {
-		return nil, errs.Wrapf(http.StatusInternalServerError, err, "error reading %s response",
-			[]interface{}{u, errs.WithMessage("Failed to parse response from /ssh/check-host endpoint")})
+		return nil, errs.Wrapf(http.StatusInternalServerError, err, "error reading %s response", u)
```

Comment:
You are changing the behavior here, I believe the err.WithMessage sets the message in the JSON, with your changes you will use the default one.